extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang

```diff
@@ -0,0 +1,686 @@
+extratrees.species.monkeypuzzle.name=智利南洋杉
+extratrees.species.rainbowgum.name=彩虹桉
+extratrees.species.pinkivory.name=象牙粉红木
+extratrees.species.blackcurrant.name=黑加仑
+extratrees.species.redcurrant.name=红加仑
+extratrees.species.blackberry.name=黑莓
+extratrees.species.raspberry.name=树莓
+extratrees.species.blueberry.name=蓝莓
+extratrees.species.cranberry.name=蔓越莓
+extratrees.species.juniper.name=杜松子
+extratrees.species.gooseberry.name=鹅莓
+extratrees.species.goldenraspberry.name=金树莓
+extratrees.species.cinnamon.name=肉桂
+extratrees.species.coconut.name=椰子
+extratrees.species.cashew.name=腰果
+extratrees.species.avacado.name=鳄梨
+extratrees.species.nutmeg.name=肉豆蔻
+extratrees.species.allspice.name=多香果
+extratrees.species.chilli.name=辣椒
+extratrees.species.staranise.name=八角
+extratrees.species.mango.name=芒果
+extratrees.species.starfruit.name=杨桃
+extratrees.species.candlenut.name=石栗
+extratrees.species.dwarfhazel.name=矮榛
+
+##################################################
+## Fruits
+
+extratrees.fruits.blackthorn=黑刺李
+extratrees.fruits.cherry_plum=樱桃李
+extratrees.fruits.peach=桃
+extratrees.fruits.nectarine=油桃
+extratrees.fruits.apricot=杏子
+extratrees.fruits.almond=扁桃
+extratrees.fruits.wild_cherry=野樱桃
+extratrees.fruits.sour_cherry=欧洲酸樱桃
+extratrees.fruits.black_cherry=黑樱桃
+extratrees.fruits.fig=无花果
+extratrees.fruits.elderberry=接骨木果
+extratrees.fruits.olive=橄榄
+extratrees.fruits.orange=橙子
+extratrees.fruits.manderin=橘子
+extratrees.fruits.tangerine=瓯柑
+extratrees.fruits.satsuma=温州蜜柑
+extratrees.fruits.key_lime=墨西哥莱檬
+extratrees.fruits.lime=酸橙
+extratrees.fruits.finger_lime=指橙
+extratrees.fruits.pomelo=柚子
+extratrees.fruits.grapefruit=西柚
+extratrees.fruits.kumquat=金橘
+extratrees.fruits.citron=枸橼
+extratrees.fruits.buddha_hand=佛手柑
+extratrees.fruits.apple=苹果
+extratrees.fruits.crabapple=海棠果
+extratrees.fruits.mango=芒果
+extratrees.fruits.pear=梨
+extratrees.fruits.osange_osange=桑橙
+extratrees.fruits.banana=香蕉
+extratrees.fruits.red_banana=红香蕉
+extratrees.fruits.platain=芭蕉
+extratrees.fruits.coffee=咖啡
+extratrees.fruits.coconut=椰子
+extratrees.fruits.cashew=腰果
+extratrees.fruits.avacado=鳄梨
+extratrees.fruits.nutmeg=肉豆蔻
+extratrees.fruits.allspice=多香果
+extratrees.fruits.chilli=辣椒
+extratrees.fruits.star_anise=八角
+extratrees.fruits.starfruit=杨桃
+extratrees.fruits.candlenut=石栗
+extratrees.fruits.hazelnut=榛子
+extratrees.fruits.butternut=灰胡桃果
+extratrees.fruits.beechnut=山毛榉坚果
+extratrees.fruits.pecan=山胡桃果
+extratrees.fruits.brazil_nut=巴西坚果
+extratrees.fruits.acorn=橡果
+extratrees.fruits.gingko_nut=白果
+extratrees.fruits.clove=丁香
+extratrees.fruits.blackcurrant=黑加仑
+extratrees.fruits.redcurrant=红加仑
+extratrees.fruits.blackberry=黑莓
+extratrees.fruits.raspberry=树莓
+extratrees.fruits.blueberry=蓝莓
+extratrees.fruits.cranberry=蔓越莓
+extratrees.fruits.juniper=杜松子
+extratrees.fruits.gooseberry=鹅莓
+extratrees.fruits.golden_raspberry=金树莓
+
+##################################################
+## Butterfly Species
+
+extrabutterflies.species.whiteadmiral.name=White Admiral
```

Comment:
_Shurg_

Reply:
Well, I tried to translate the names of butterflies. I failed completely before I finished it.

Reply:
Maybe need an expert in biology especially in lepidopterology.

Reply:
Ignore all of them then...